app.py

```diff
@@ -70,10 +70,13 @@ def _swizzle_RequestHandler_prepare2(self):
     _super_RequestHandler_prepare3 = tornado.web.RequestHandler.prepare
     def _swizzle_RequestHandler_prepare3(self):
         self.request.is_low_bandwidth = 0
+        self.request.is_mobile = 0
         if "User-Agent" in self.request.headers:
             ua = user_agents.parse(self.request.headers["User-Agent"])
             if ua.is_mobile or ua.is_tablet:
                 self.request.is_low_bandwidth = 1
+            if ua.is_mobile:
+                self.request.is_mobile = 1
```

Comment:
It's more reliable to do this with CSS media queries instead of UA-sniffing.  Put a media block like this inside .sign_view and you can apply style to any browsers that are resized small enough:

```less
media screen and (max-width:@hides_under_mobile_threshold) {
  ...
}
```